ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -19,8 +19,8 @@ distro_custom() {
 
     # Use a more recent python version for unit testing, this allows us to also test installing
     # pydaos into virtual environments.
-    # TODO: el8 should have python3.11 available but our CI does not for some reason.
     dnf -y install python39 python39-devel
+    dnf -y install python3.11 python3.11-devel
```

Comment:
Are these both needed?

Reply:
Yes.  I would expect the `devel` package to bring in the core package but this line installs both of them, and I wouldn't want to give the impression that `python3.11` was already installed and this was only bringing in the `devel` add-on.

Reply:
Sorry.  I see my question was unclear.  Are both `python3{9,.11}{,-devel}` needed.  I.e. do we need both of those Python versions installed?  Or should the 3.11 be instead of the 39 rather than in addition to it?

Reply:
Ah, good point.  No, having re-read the PR I don't think 39 is needed.  At one point in this PR 39 was the latest that would work but with the el8.8 fix that changed to 3.11 and I didn't remove 39 at that time.